.editorconfig

```diff
@@ -0,0 +1,20 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
+quote_type = single

```

Comment:
@vinibeloni Have you checked there is only single quote used across all code?

Reply:
The idea at first is to bring the main properties of the Editorconfig applying a convention since the project has not followed a linear pattern.

In this case, when I searched for double quotes were found more single than double.